cmd/meroxa/root/apps/apps.go

```diff
@@ -58,11 +87,9 @@ func (*Apps) SubCommands() []*cobra.Command {
 		builder.BuildCobraCommand(&Describe{}),
 		builder.BuildCobraCommand(&Init{}),
 		builder.BuildCobraCommand(&List{}),
-		builder.BuildCobraCommand(&Logs{}),
```

Comment:
I'd imagine the idea is to remove this for now until we have a better way to pull logs from mdpx?

Reply:
Yup!!